### PR TITLE
Fix issues with python 2.7

### LINKED ISSUE #1366: Ubuntu error: cannot import name loop
Hi, 
here is something I am experiencing on ubuntu only. This is working on Mac os.
I have a user that experience it also (Ubuntu 16.04).
```
flat schema tag detected - switching to flat input schema
Exception ImportError: 'No module named abc' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
```

Any ideas about this ? do we have to update something ?

thx

<details>
               :
           .CL:CC CC             _Q     _Q  _Q_Q    _Q    _Q              _Q
         CC;CCCCCCCC:C;         /_\)   /_\)/_/\\)  /_\)  /_\)            /_\)
         CCCCCCCCCCCCCl       __O|/O___O|/O_OO|/O__O|/O__O|/O____________O|/O__
      CCCCCCf     iCCCLCC     /////////////////////////////////////////////////
      iCCCt  ;;;;;.  CCCC
     CCCC  ;;;;;;;;;. CClL.                          c
    CCCC ,;;       ;;: CCCC  ;                   : CCCCi
     CCC ;;         ;;  CC   ;;:                CCC`   `C;
   lCCC ;;              CCCC  ;;;:             :CC .;;. C;   ;    :   ;  :;;
   CCCC ;.              CCCC    ;;;,           CC ;    ; Ci  ;    :   ;  :  ;
    iCC :;               CC       ;;;,        ;C ;       CC  ;    :   ; .
   CCCi ;;               CCC        ;;;.      .C ;       tf  ;    :   ;  ;.
   CCC  ;;               CCC          ;;;;;;; fC :       lC  ;    :   ;    ;:
    iCf ;;               CC         :;;:      tC ;       CC  ;    :   ;     ;
   fCCC :;              LCCf      ;;;:         LC :.  ,: C   ;    ;   ; ;   ;
   CCCC  ;;             CCCC    ;;;:           CCi `;;` CC.  ;;;; :;.;.  ; ,;
     CCl ;;             CC    ;;;;              CCC    CCL
    tCCC  ;;        ;; CCCL  ;;;                  tCCCCC.
     CCCC  ;;     :;; CCCCf  ;                     ,L
      lCCC   ;;;;;;  CCCL
      CCCCCC  :;;  fCCCCC
       . CCCC     CCCC .
        .CCCCCCCCCCCCCi
           iCCCCCLCf
            .  C. ,
               :
 Exception ImportError: 'No module named abc' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
 Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
[...x500...]
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
flat schema tag detected - switching to flat input schema

Status: Cyclus run successful!
Output location: cyclus.sqlite
</details>

### PR DESCRIPTION
**Pull request by adityapb**
Fixes #1366 

The cyclus.lazyasd module was probably reused from another project. A lot of python 3 specific stuff wasn't being used by cyclus so I just removed it. Not sure if it is a good idea. @scopatz

### CODE REVIEW

File: cyclus/lazyasd.py [hunk 1]
```diff
@@ -7,8 +7,6 @@
 import builtins
```
Comment (scopatz) [inline cyclus/lazyasd.py]: Please remove this entire file.

File: cyclus/system.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
 
 @lazyobject
```
Comment (scopatz) [inline cyclus/system.py]: All of these lazyobject decorators will need to be removed. All of the functions here that are decorated with lazyobject will need to be rewritten.

File: cyclus/system.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
```
Comment (scopatz) [inline cyclus/system.py]: This should be 

```python
PY_VERSION_TUPLE = sys.version_info[:3]
```

File: cyclus/system.py [hunk 4]
```diff
@@ -12,7 +12,7 @@ def PY_VERSION_TUPLE():
 
 @lazyobject
 def curio():
```
Comment (scopatz) [inline cyclus/system.py]: This function should be replaced by:

```python
if PY_VERSION_TUPLE < (3, 5, 2):
   curio = None
else:
    try:
        import curio
    except ImportError:
        curio = None
```

File: cyclus/system.py [hunk 5]
```diff
@@ -23,7 +23,7 @@ def curio():
 
 @lazyobject
 def asyncio():
```
Comment (scopatz) [inline cyclus/system.py]: This should look like the curio replacement above, but replace curio with asyncio.

File: cyclus/system.py [hunk 6]
```diff
@@ -34,7 +34,7 @@ def asyncio():
 
 @lazyobject
 def websockets():
```
Comment (scopatz) [inline cyclus/system.py]: This should look like the curio replacement above, but replace curio with websockets.

File: cyclus/system.py [hunk 7]
```diff
@@ -45,7 +45,7 @@ def websockets():
 
 @lazyobject
 def concurrent_futures():
```
Comment (scopatz) [inline cyclus/system.py]: This should look like the curio replacement above, but replace curio with `concurrent_futures`.

**Comment by scopatz** [comment]
Thanks for putting this in @adityapb! See comments in-line

**Comment by bam241** [comment]
can still see some of them"
```
Exception ImportError: 'No module named builtins' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
Exception ImportError: 'cannot import name loop' in 'eventhooks.eventloophook' ignored
```

**Comment by adityapb** [comment]
builtins module is a part of future package. `pip install future` should fix it

**Comment by adityapb** [comment]
I am removing the file anyway, so doesn't matter

**Comment by scopatz** [comment]
Also @adityapb, let us know in the comments here when you have an update!

File: cyclus/system.py (hunk 4 above)
Comment (adityapb) [inline cyclus/system.py]: Aren't these functions supposed to return an instance of the module they are importing?

Comment (scopatz) [inline cyclus/system.py]: Yes, but if we aren't having them lazily created, the functions should be removed and the imports should happen at the module level, non-lazily.

Comment (adityapb) [inline cyclus/system.py]: Ah! So I get rid of the functions altogether. Got it! Thanks!

**Comment by adityapb** [comment]
I've made the changes @scopatz

**Comment by scopatz** [comment]
LGTM, Thanks!

**Comment by bam241** [comment]
Yes it works fine
thx @adityapb for putting this !
thx @scopatz for the reviews !

**Pull request merged**